README.md

```diff
@@ -1,85 +1,135 @@
 # tar_scm (OBS source service) [![Build Status](https://travis-ci.org/openSUSE/obs-service-tar_scm.png?branch=master)](https://travis-ci.org/openSUSE/obs-service-tar_scm)
 
-This is an [Open Build Service](http://openbuildservice.org/) source
-service. It uses an SCM client to checkout or update a package by
-creating a source tarball from a source code repository.
+This is the git repository for
+[openSUSE:Tools/obs-service-tar_scm](https://build.opensuse.org/package/show/openSUSE:Tools/obs-service-tar_scm),
+which provides several [source
+services](http://openbuildservice.org/help/manuals/obs-user-guide/cha.obs.source_service.html)
+for the [Open Build Service](http://openbuildservice.org/) which all
+assist with packaging source code from SCM (source code management)
+repositories into tarballs.  The authoritative source is
+https://github.com/openSUSE/obs-service-tar_scm.
+
+## Services
+
+### tar_scm *(deprecated)*
+
+`tar_scm` is the legacy source service used to create a source tarball
+from one of the supported SCM (source code management) tools: `git`,
+`hg`, `svn`, and `bzr`.
 
-It supports many options, e.g. it can adjust resulting tarball
-parameters, include or exclude particular files when creating this
-tarball or generate an rpm changelog from the SCM commit log. For the
-full list of options please see tar_scm.service.in.
+`tar_scm` supports many options, e.g. it can adjust resulting tarball
+parameters, include or exclude particular files when creating the
+tarball, or generate an `rpm` changelog from the SCM commit log. For the
+full list of options please see `tar_scm.service.in`.
 
-Apart from various SCM like git, hg, bzr or svn it additionally
+Apart from various SCM like git, hg, bzr or svn, it additionally
 supports `--url` option that allows you to specify URL of the upstream
 tarball to be downloaded.
 
-This is the git repository for
-[openSUSE:Tools/obs-service-tar_scm](https://build.opensuse.org/package/show/openSUSE:Tools/obs-service-tar_scm).
-The authoritative source is
-https://github.com/openSUSE/obs-service-tar_scm
-
-The service can be used in combination with other services like
+`tar_scm` can be used in combination with other services like
 [download_files](https://github.com/openSUSE/obs-service-download_files),
```

Comment:
yes, but they don't have any interaction, so why mention it here?

Reply:
Fair point ... OTOH promoting awareness of available source services seems like a good thing.  Is there some central index of source services?  If not, we should create one, and then this README can just say "for other source services, see this index".

Reply:
osc api /service

shows all services including documentation .....

Reply:
That's an awesome tip! But how is anyone supposed to find out about it?

Reply:
It also assumes that you have direct access to `osc`, whereas documentation can be read even on a phone ...

Reply:
yes, we used to have a webui interface showing all options with an editor showing text lines, combo boxes and had drag and drop for ordering. It got dropped unfortunatly during some refactoring.

Reply:
Ah, that's a shame - that sounded really nice. I hope it comes back at some point!!